examples/chart/teleport-demo/docker/cloudflare-agent/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+docker pull gcr.io/kubeadm-167321/cloudflare-agent:latest
```

Comment:
Best practice is not to use `latest` images, because it leads to unpredictable builds. Use tags if it is possible.

Reply:
Also, the whole script does not make sense. You pull image and at the same time rebuild it and push again.

Reply:
My intention was to pull all existing layers from the repository to fill the layer cache, then rebuild in the hope that Docker would use the cache that had just been pulled for a faster build.

Reading about it though, it seems that I might need the `--cache-from` argument to `docker build` in order to do this. I'll check it out.

Reply:
Updated this now to use a version tag on all Docker images as well as setting `--cache-from` properly.